apps/arena/lib/arena/game/player.ex

```diff
@@ -375,7 +380,9 @@ defmodule Arena.Game.Player do
   end
 
   defp apply_skill_cooldown(player, skill_key, %{cooldown_mechanism: "time", cooldown_ms: cooldown_ms}) do
-    put_in(player, [:aditional_info, :cooldowns, skill_key], cooldown_ms)
+    cooldown_multiplier = player |> get_golden_clock_effect |> get_cooldown_multiplier
```

Comment:
Why not having something similar to what the stamina is doing? have a field in the player additional info that determine a cooldown multiplier and change that when you gain the `golden_clock_effect`

Reply:
As we talked, we can do something similar to what `apply_stat_modifier` is doing when it has to reduce the stamina interval, and use that with another field in the player called `cooldown_multiplier`

Reply:
fixed